ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -116,6 +116,14 @@
       arity = "1")
   private Integer downstreamHttpPort;
 
+  @SuppressWarnings("FieldMayBeFinal")
+  @Option(
+      names = {"--downstream-http-path"},
+      description = "The path to which received requests are forwarded",
```

Comment:
Think we should Include the default value

Reply:
done